backend/api/Services/RobotService.cs

```diff
@@ -48,6 +48,20 @@ public async Task<Robot> Create(CreateRobotQuery newRobot)
                 videoStreams.Add(videoStream);
             }
 
+            var availableDecks = new List<AssetDeck>();
+            foreach (var assetDeckQuery in newRobot.AvailableDecks)
+            {
+                var defaultPose = new Pose();
+                assetDeckQuery.DefaultPose.CopyIsarPoseToRobotPose(defaultPose);
+
+                var assetDeck = new AssetDeck
+                {
+                    DeckName = assetDeckQuery.DeckName,
+                    DefaultPose = defaultPose
+                };
+                availableDecks.Add(assetDeck);
+            }
+
```

Comment:
is it smarter to write a function that takes in AvailableDecks and converts to a list of asset decks in the constructor?